src/Compilers/VisualBasic/Test/Emit/Emit/OptionalArgumentsTests.vb

```diff
@@ -814,7 +814,7 @@ Class P
         Report(C.F6())
     End Sub
     Shared Sub Report(o As Object)
-        Dim value As Object = If (TypeOf o is Date, DirectCast(o, Date).ToString("MM/dd/yyyy HH:mm:ss"), o)
+        Dim value As Object = If (TypeOf o is Date, DirectCast(o, Date).ToString("yyyy-MM-dd HH:mm:ss"), o)
```

Comment:
What is this change?

Reply:
Slashes are, by specification, formatted in a locale-specific way for dates. I'm making the test not be locale-specific.

---
In reply to: [202467195](https://github.com/dotnet/roslyn/pull/27312#discussion_r202467195) [](ancestors = 202467195)